accio-sqlrewrite/src/test/resources/tpch_mdl.json

```diff
@@ -291,6 +291,27 @@
       ]
     }
   ],
+  "cumulativeMetrics": [
+    {
+      "name": "WeeklyRevenue",
+      "baseModel": "Orders",
+      "measure": {
+        "name": "totalprice",
+        "type": "int4",
+        "expression": "sum(totalprice)"
```

Comment:
Is this field required `"expression": "sum(totalprice)"` ?

Reply:
Yes, `name` is a column name for this metric but the expression will mention which aggregation it used.

Reply:
hmmm, IIRC I didn't find we use the expression in getCumulativeMetricSql.

Reply:
ohhh.. I missed this part. I just do `sum` everything. it should support other aggregation function.

Reply:
I think the filed should be
```
"measure": {
  "name": "totalprice",
   "type": "int4",
   "operator": "sum",
   "refColumn": "totalprice"
}
```

Reply:
Sure, looks good to me.